### DISCUSSION TITLE
Weblate translations are temporarily locked

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#267] [Announcements]
Weblate translations are locked until [this issue](https://github.com/WeblateOrg/weblate/issues/13300#issuecomment-2561820303) is resolved.

This should be temporary but just in case you are looking to contribute translations or fix existing translations, please use Github's edit file feature. Click [here](https://github.com/FossifyOrg/General-Discussion?tab=readme-ov-file#how-can-i-suggest-an-edit-to-a-file) for instructions. 

Thank you!

**Comment by ltguillaume** [comment]
I see hundreds of corrupted Dutch strings now, where `"Something` has been replaced with ` 	omething`.

**Reply by naveensingh** [reply]
On just weblate or here as well?

**Reply by ltguillaume** [reply]
Ah, seems like it's on Weblate only.

**Comment by naveensingh** [comment]
Seems to be fixed now 🎉
